kubemarine/procedures/install.py

```diff
@@ -202,12 +202,35 @@ def system_prepare_policy(group: NodeGroup):
     if kubernetes.is_cluster_installed(cluster) and audit_config is True and cluster.context['initial_procedure'] != 'add_node':
         for control_plane in collect_node:
             config_new = (kubernetes.get_kubeadm_config(cluster.inventory))
+
+            # we need InitConfiguration in audit-on-config.yaml file to take into account kubeadm patch for apiserver
+            init_config = {
+                'apiVersion': group.cluster.inventory["services"]["kubeadm"]['apiVersion'],
+                'kind': 'InitConfiguration',
+                'localAPIEndpoint': {
+                    'advertiseAddress': control_plane['internal_address']
+                },
+                'patches': {
+                    'directory': '/etc/kubernetes/patches'
+                }
+            }
+
+            config_new = config_new + "---\n" + yaml.dump(init_config, default_flow_style=False)
```

Comment:
Here we assume patches with `bind-address` are already on node. This assumption is not true for clusters installed with older kubemarine. Need to make sure patches are on nodes

Reply:
fixed